cli.sh

```diff
@@ -2160,7 +2311,15 @@ config_search_dir() {
 #   global Git configuration.
 #####################################################
 config_global_shared_hook_repos() {
+
     if [ "$1" = "set" ]; then
+
+        SHARED_TYPE="--global"
+        if echo "$2" | grep -qE "\-\-(local)"; then
```

Comment:
Out of curiosity, why do this a regex?

Reply:
Dont know :-) This ist stupid kind of, yes :-) Good catch